deps/oob-1.0.2/jsonutil.c

```diff
@@ -404,7 +410,35 @@ void setoobConnectioninfo(const char* host,
     copyString("https", connectionInfo.protocol, 8);
   else
     copyString("http", connectionInfo.protocol, 8);
+}
 
+void setoobDsninfo(const char *key, const char *val) {
```

Comment:
How do you ensure that you don't repeatedly add the same key for multiple connections ?